### PR TITLE
uboot-mvebu: add missing OpenSSL compat patches

### PR DESCRIPTION
**Pull request by ratkaj**
Add missing OpenSSL compatibility patches. Fixes the following build
issue: "undefined reference to `EVP_MD_CTX_create'".

Add missing UBOOT_MAKE_FLAGS variable.
Fixes "/bin/sh: HOSTCPPFLAGS: command not found" build errors

Fix "SETEXPR" redefinition warning

Signed-off-by: Marko Ratkaj <marko.ratkaj@sartura.hr>

### CODE REVIEW

**Comment by diizzyy** [comment]
I'm probably missing something here but we don't have (at least yet) OpenSSL 1.1+ in tree?

File: package/boot/uboot-mvebu/Makefile [hunk 1]
```diff
@@ -16,6 +16,8 @@ PKG_HASH:=f54baf3f9325bf444c7905f3a5b6f83680edb1e6e1a4d5f8a5ad80abe885113f
 include $(INCLUDE_DIR)/u-boot.mk
 include $(INCLUDE_DIR)/package.mk
 
+UBOOT_MAKE_FLAGS:=
+
```
Comment (hauke) [inline package/boot/uboot-mvebu/Makefile]: Why is this needed?

**Comment by hauke** [comment]
It would make sense if you split this into multiple patches.
One patch with the backports needed to fix the openssl 1.1.x problems
One patch for the SETEXPR fix
and the for the other problem.

File: package/boot/uboot-mvebu/Makefile (hunk 1 above)
Comment (ratkaj) [reply package/boot/uboot-mvebu/Makefile]: Douring build we get more than 60 errors:
/bin/sh: HOSTCPPFLAGS: command not found

Above change will remove those errors. Several other uboot Makefiles use this, examples are uboot-ar71xx and uboot-lantiq

**Comment by ratkaj** [comment]
@diizzyy sry for the late response
we are using 1.0.2L and "L" is where the problems started. These patches will add compatibility with 1.1.x and fix the issues. Maybe we can make make a smaller patch just for 1.0.2l but i dont have time to debug that at the moment

**Comment by ratkaj** [comment]
@hauke splitted into 3 patches

**Comment by hauke** [comment]
Thank you for your patches, they are applied to master now.
